CLAUDE.md

```
@@ -1,18 +1,23 @@
 
 ## Project Context
-This repository contains Behat step definitions for Drupal projects, organized as traits that can be included in Behat contexts. The steps provide reusable testing functionality for Drupal components and features.
+This repository contains Behat step definitions for PHP projects (with specialized support for Drupal), organized as traits that can be included in Behat contexts. The steps provide reusable testing functionality for generic PHP components and Drupal-specific features.
 
 ## Source files location
 
 Source files are located in the `src` directory. Each trait is organized into a separate file, and the steps are defined within those files.
 
 
-## Installation & Requirements
+## Installation & Requirements for cosnuming this library
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)